images/build_images

```diff
@@ -54,43 +63,63 @@ def main():
     parser.add_argument('-p', '--push', help='Push Docker images after building', action='store_true')
     args = parser.parse_args()
 
-    image_folder = Path(Path(__file__).parent, 'cloudera_service').resolve()
-
     if args.dry_run:
         logger.info('Doing dry-run of tool ...')
 
     sdc_version = _get_sdc_version(PARCEL_MANIFEST_URL_TEMPLATE.format(build=args.sdc_build,
                                                                        s3_bucket=args.sdc_s3_bucket))
+
+    images = _get_sdc_images(args, sdc_version)
+    _build_push_images(args, images, sdc_version)
+
+
+def _build_push_images(args, images, sdc_version):
+    image_folder = Path(Path(__file__).parent, 'cloudera_service').resolve()
```

Comment:
I think you can use `/` to join `Path` elements instead of having to nest Path like this. Also, keep in mind `path` was only introduced in Python 3.4 so let's just be sure there's no where that we document supporting earlier versions of Python (and, if we do, that we update it)

Reply:
Done.